virt/learn_more_about_openshift_virtualization.adoc

```diff
@@ -0,0 +1,108 @@
+include::modules/virt-document-attributes.adoc[]
+[id="learn_more_about_openshift_virtualization"]
+= Learn more about {VirtProductName}
+:context: learn_more_about_openshift_virtualization
+
+toc::[]
+
+Use the following sections to find content to help you learn about and use {VirtProductName}.
```

Comment:
I think this is a good start to making our docs more accessible, but it's just linking into what we already have.
The next step needs to be having complete workflows, similar to what the integration team is doing in their blogs.

Reply:
+1 - Yes, we need more comprehensive workflows. Right now, however, I think this page does help in grouping together topics that otherwise remain scattershot in the TOC. I will update my project document and address this when we talk more about personas and will invite Andrew Sullivan to the conversation as well. I would eventually like to use this page as a blueprint to restructure our TOC around workflow and personas, but that will require some planning as well as a test bed.